apps/web/src/pages/AnnouncementsPage/SitewideAnnouncementForm.tsx

```diff
@@ -0,0 +1,216 @@
+import * as React from "react";
+import { FormProvider, ValidateResult, useForm } from "react-hook-form";
+import { defineMessage, useIntl } from "react-intl";
+
+import {
+  formDateTimeStringToDate,
+  nowUTCDateTime,
+} from "@gc-digital-talent/date-helpers";
+import {
+  Scalars,
+  SitewideAnnouncement,
+  SitewideAnnouncementInput,
+} from "@gc-digital-talent/graphql";
+import {
+  Input,
+  RichTextInput,
+  Submit,
+  SwitchInput,
+} from "@gc-digital-talent/forms";
+import {
+  commonMessages,
+  errorMessages,
+  formMessages,
+} from "@gc-digital-talent/i18n";
+import { Button, ToggleSection } from "@gc-digital-talent/ui";
+
+import labels from "./labels";
+
+const invalidDateTimeMessage = defineMessage({
+  defaultMessage: "Enter the date in the form yyyy-MM-dd HH:mm:ss",
```

Comment:
```suggestion
  defaultMessage: "Enter the date in the format yyyy-MM-dd HH:mm:ss",
```

Reply:
OK, but I might ask you to help edit the French string that comes back, too.  :grin:

Reply:
It looks like the translator chose to use the work "format" already, which I guess is the same as in English?

Reply:
> It looks like the translator chose to use the work "format" already, which I guess is the same as in English?

Yes, it it.